tests/test_modeling_common.py

```diff
@@ -3708,6 +3712,12 @@ def get_mean_reldiff(failcase, x, ref, atol, rtol):
                                     # Otherwise fails for e.g. WhisperEncoderModel
                                     if "attention_mask" in inspect.signature(model_eager.forward).parameters:
                                         processed_inputs["attention_mask"] = dummy_attention_mask
+                                    # Otherwise fails for e.g. CLIPModelTest
+                                    if (
+                                        "pixel_values" in inspect.signature(model_eager.forward).parameters
+                                        and model_class.__name__ == "CLIPModel"
```

Comment:
We can probably just remove the CLIP condition here - I'd expect it to be true for all vision models 

```suggestion
```

Reply:
Kept it this way because other vision-language models likely don't support SDPA yet (even though they support flash attention).

Reply:
That's covered in the `_supports_sdpa` check at the start of the test. Removing the clip condition means that as soon as sdpa support is added to other vision models they're (almost certainly) be compatible for this part of the test

Reply:
Done.